R/plotNorm.R

```diff
@@ -76,25 +80,141 @@ plotNorm <- function(DGEdata,
 
     tall <- tall %>%
         rbind(tall_tmm)
+    title <- stringr::str_c("Log2CPM before/after", normalize, "normalization", sep = " ")
 
-    if (tolower(plotType) == "density") {
-        resultPlot <- ggplot(tall, aes(x = Log2CPM, color = SampleID)) +
-            geom_density() +
-            facet_grid(~Normalization) +
-            ggtitle(stringr::str_c("Log2CPM before/after", normalize, "normalization", sep = " "))  +
-            theme_gray() +
-            theme(legend.position = "none")
+    if (tolower(plotCategory) == "density") {
```

Comment:
poltType check is a major functionality and I would recommend keeping Cx code in one section and ggplot in another. It would be great if we keep plotType check outside and plotCategary check inside.

Reply:
done